content/realtime/channel-params.textile

```diff
@@ -20,12 +18,38 @@ The methods provided for specifying channel parameters, and the currently availa
 
 h2(#supported-params). Currently supported channel params
 
-- rewind := Used to request that an attachment start from some number of messages or point in time in the past. See "rewind":#rewind for more information
-- delta := **In an experimental state**. Used to request that data payloads should be sent as deltas to the previous payload. "Contact us":https://www.ably.io/contact for more information and supported values
+- rewind := Used to request that an attachment start from some number of messages or point in time in the past. See "rewind":#./rewind for more information
+- delta := Used to request that data payloads should be sent as deltas to the previous payload. See "delta":./delta for more information
+
+h2(#using-params-with-lib-v12). Using channel params with v1.2 or later Ably client libraries
+
+Channel params may be specified in the @ChannelOptions@ when obtaining a @Channel@. A collection of channel params is expressed as a map of string key/value pairs. The @ChannelOptions@ associated with a channel may also be updated by calling "setOptions":./channel#setOptions. The params associated with a channel take effect when the channel is first attached; if the params are subsequently modified via a call to "setOptions":./channel#setOptions, then that call triggers attach operation that applies the updated params, if successful.
```

Comment:
Clicking the 'setOptions' link I get:

> File not found: /realtime/channel-params/channel

when viewing a local build (via `bundle exec nanoc compile && bundle exec nanoc view -p 4000`) at `http://localhost:4000/realtime/channel-params/channel#setOptions`

The Heroku preview presents that as:

> 404 Not Found : nginx